corehq/apps/sms/static/sms/js/languages.js

```diff
@@ -1,12 +1,15 @@
-hqDefine('sms/js/languages', function() {
-    var initialPageData = hqImport('hqwebapp/js/initial_page_data');
+hqDefine('sms/js/languages',[
+    "jquery",
+    "hqwebapp/js/initial_page_data",
+    "app_manager/js/supported_languages",
+], function($, initialPageData, supportedLanguages) {
     $(function () {
         var langs = initialPageData.get('sms_langs');
         var saveURL = initialPageData.reverse("edit_sms_languages");
         var edit = true;
         var validate = true;
-        var SupportedLanguages = hqImport('app_manager/js/supported_languages').SupportedLanguages;
-        var sl = new SupportedLanguages({
+        var SupportedLanguages = supportedLanguages.SupportedLanguages;
```

Comment:
seems a little odd to have this as a var definition instead of just referencing it directly